core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -750,6 +754,25 @@ public Put withCondition(MutationCondition condition) {
     return (Put) super.withCondition(condition);
   }
 
+  /**
+   * Returns whether this Put is a blind write.
+   *
+   * @return whether this Put is a blind write
+   */
+  public boolean isBlind() {
```

Comment:
I might miss something, so just a question.
Don't we need these methods in `Delete`?

Reply:
I don't think we need the `blind` option for `Delete` because we always need the pre-read for `Delete`. What do you think?